src/Entity/Board.php

```diff
@@ -451,4 +451,14 @@ public function setHearingPostResponseTemplate(?MailTemplate $hearingPostRespons
 
         return $this;
     }
+
+    public function getFinalStatus(): string
+    {
+        $rawPlaces = explode(
+            PHP_EOL,
+            trim($this->getStatuses())
+        );
+
+        return trim(end($rawPlaces));
```

Comment:
`end` returns `false` if the array is empty (cf. https://www.php.net/manual/en/function.end.php#refsect1-function.end-returnvalues). You should probably handle this somehow, e.g. with

```php
return end($rawPlaces) ?: '';
```

Reply:
👍